notifier/api/flask/endpoints.py

```diff
@@ -0,0 +1,21 @@
+import json
+from flask import Blueprint
+from ...core.apiimpl import ApiImpl
+
+api = ApiImpl()
+endpoint = Blueprint("endpoint", "Notifier")
```

Comment:
These lines are no consistent with Celery API. If we use Flask directly we can use it here instancing the FlaskApp class (as in the Celery API implementation). Moreover, the object `endpoint` cannot use that name because this is the name of the file